components/EditForm/index.js

```diff
@@ -13,8 +13,22 @@ export default function EditForm({ project, setEditMode }) {
   const {
     register,
     handleSubmit,
+    control,
     formState: { errors },
-  } = useForm();
+  } = useForm({
+    defaultValues: {
+      material: project.material.map((material) => ({
```

Comment:
would it be okay to take this into the next PR? Because changing this would mean to change every file, where the projects are used and also changing the data set manually in mongo DB. At the moment our group is not complete, so we don't want to decide without the rest of the group.

Reply:
okay we updated it now.